hpx/runtime/threads/threadmanager_impl.hpp

```diff
@@ -296,6 +296,12 @@ namespace hpx { namespace threads
             pool_.reset_thread_distribution();
         }
 
+	// Returns the underlying scheduling policy
+        scheduling_policy_type& get_pool_sched() const
+	{
+	    return pool_.get_sched();
+	}
+
```

Comment:
We need to know the number of disabled/suspended threads and it is used here:
https://github.com/allscale/allscale_runtime/blob/multi_objective_optimizer/src/components/scheduler_component.cpp#L237